### PR TITLE
button persists!

### PR DESCRIPTION
**Pull request by easherma**
Fixes # (issue)

https://github.com/cityofaustin/techstack/issues/3029

# Testing Notes

Try making/unmaking buttons in service page steps. 

https://joplin-pr-3029-persist-button.herokuapp.com/

# Checklist:
- [x] Request reviewers
- [x] Slack-out message for review
- [x] Link this PR in the issue
- [ ] Moved card to *review* in zenhub

### CODE REVIEW

File: joplin/base/wagtail_hooks.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 from html.parser import HTMLParser
 
 import wagtail.admin.rich_text.editors.draftail.features as draftail_features
-from wagtail.admin.rich_text.converters.html_to_contentstate import BlockElementHandler
+from wagtail.admin.rich_text.converters.html_to_contentstate import BlockElementHandler, InlineStyleElementHandler
```
Comment (briaguya) [inline joplin/base/wagtail_hooks.py]: Where are we using `InlineStyleElementHandler`?

File: joplin/base/wagtail_hooks.py [hunk 2]
```diff
@@ -226,7 +227,7 @@ def register_button_feature(features):
     )
 
     features.register_converter_rule('contentstate', feature_name, {
-        'from_database_format': {'div.rich-text-button-link': BlockElementHandler(type_)},
+        'from_database_format': {tag: BlockElementHandler(type_)},
```
Comment (easherma) [inline joplin/base/wagtail_hooks.py]: this fixes the bug

File: joplin/css/editor.scss [hunk 3]
```diff
@@ -71,3 +71,28 @@ label {
 .coa-placeholder-elm {
     color: $coa-color-dark-gray2 !important;
 }
+
```
Comment (easherma) [inline joplin/css/editor.scss]: this is gravy

File: joplin/base/wagtail_hooks.py (hunk 1 above)
Comment (easherma) [reply joplin/base/wagtail_hooks.py]: ahh we aren't atm, i'll remove it (i was playing around with something else)

**Pull request merged**